mica-search/src/main/java/org/obiba/mica/search/csvexport/generators/StudiesCsvReportGenerator.java

```diff
@@ -91,8 +94,13 @@ protected void writeEachLine(CSVWriter writer) {
 
     line.add(studySummaryDto.getAcronym(0).getValue());
     line.add(studySummaryDto.getName(0).getValue());
-    if (mustShow("showStudiesDesignColumn"))
-      line.add(studySummaryDto.getDesigns(0));
+
+    if (mustShow("showStudiesDesignColumn")) {
+      String studyDesign = studySummaryDto.getDesigns(0);
```

Comment:
There is still a possible dto error. You should getDesignsCount() before getting the first one.